release_args.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# this function will be sourced from release.sh and be called from release_functions.sh
+update_versions_modify_files() {
+  newReleaseVersion="${1}"
+  valuesYAML=k8s/helm/values.yaml
+  componentPatchTplYAML=k8s/helm/component-patch-tpl.yaml
+
```

Comment:
```suggestion

echo "Set image in values.yaml and component patch template"
```
just some output for tracebility